### PR TITLE
Organize smart contracts section

### PR DESCRIPTION
**Pull request by orkunkl**
Resolves https://github.com/CosmWasm/cawesome-wasm/issues/1

### CODE REVIEW

**Comment by orkunkl** [comment]
In [cosmwasm/packages](https://github.com/CosmWasm/cosmwasm/tree/master/packages) I see 4 packages although in this repo there is only `cw-storage` is mentioned. @ethanfrey do you want me to add them as well

**Comment by ethanfrey** [comment]
For the end user, they should likely never use vm. they will only use schema in some auto-generated example code.

The important ones are std and storage

File: README.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ Please read the [contribution guidelines](CONTRIBUTING.md) if you want to contri
...
+  - [Smart Contracts](#smart-contracts)
+  - [Smart Contract Libraries](#smart-contract-libraries)
+  - [Tooling](#tooling)
+  - [dApps](#dapps)
 
 ## General Resources
 
-- [CosmWasm official documentation](https://www.cosmwasm.com)
+- [CosmWasm official documentation](https://docs.cosmwasm.com)
 - [CosmWasm framework](https://github.com/confio/cosmwasm) - you will import this for your contracts, README has useful info and links
```
Comment (ethanfrey) [inline README.md:18]: Renamed to `CosmWasm/cosmwasm` - same as all other `confio/` packages

File: README.md [hunk 2]
```diff
@@ -29,16 +29,37 @@ guidelines on
 [sharing code reviews in a decentralized manner](https://github.com/confio/cosmwasm-template/blob/master/Importing.md)
 which can be used as the basis for a peer-reviewed audit.
 
-- [cw-escrow](https://crates.io/crates/cw-escrow) by [ethanfrey](https://github.com/ethanfrey) ([repo](https://github.com/confio/cosmwasm-examples/tree/master/escrow)): A simple CosmWasm contract for an escrow with arbiter and timeout
-- [cw-erc20](https://crates.io/crates/cw-erc20) by [webmaster128](https://github.com/webmaster128) ([repo](https://github.com/confio/cosmwasm-examples/tree/master/erc20)): Porting the erc20 token standard to CosmWasm
+### Official Contracts
+
+Official contracts are developed and maintained by cosmwasm team
```
Comment (ethanfrey) [inline README.md:34]: I would say "Official _example_ contracts are developed and maintained by cosmwasm team to serve as best practices and learning references"

File: README.md [hunk 3]
```diff
@@ -29,16 +29,37 @@ guidelines on
...
+
+- [cw-escrow](https://crates.io/crates/cw-escrow) by [ethanfrey](https://github.com/ethanfrey) ([repo](https://github.com/confio/cosmwasm-examples/tree/master/escrow)): A simple CosmWasm contract for an escrow with arbiter and timeout.
+- [cw-erc20](https://crates.io/crates/cw-erc20) by [webmaster128](https://github.com/webmaster128) ([repo](https://github.com/confio/cosmwasm-examples/tree/master/erc20)): Porting the erc20 token standard to CosmWasm.
+
+### Plus Contracts
```
Comment (ethanfrey) [inline README.md:39]: Nice to list the specs as well... mainly the READMEs. These give a nice analogy to the ERC20, ERC721, etc specs people like to look for

File: README.md (hunk 2 above)
Comment (ethanfrey) [reply README.md:34]: Wait... these same contracts are listed below as "trivial contracts".

Maybe best just to remove this section and add the comment about "demonstrate best practices and learning references" to the "Trivial Contracts" section

File: README.md [hunk 4]
```diff
@@ -29,16 +29,31 @@ guidelines on
...
+- [name service](https://github.com/CosmWasm/cosmwasm-examples/tree/master/nameservice): Blockchain name service in cosmwasm.
+- [erc20](https://github.com/CosmWasm/cosmwasm-examples/tree/master/erc20): Implementation of Ethereum's [ERC20](https://eips.ethereum.org/EIPS/eip-20) interface.
+- [escrow](https://github.com/CosmWasm/cosmwasm-examples/tree/master/escrow): This is a simple single-use escrow contract.
 
 ## Smart Contract Libraries
 
-- [cw-storage](https://crates.io/crates/cw-storage) by [ethanfrey](https://github.com/ethanfrey) ([repo](https://github.com/confio/cosmwasm-examples/tree/master/escrow)): Helper methods to reduce boilerplate for storing data types. Easier and more secure persistence layer.
+- [cw-std](https://github.com/CosmWasm/cosmwasm/tree/master/packages/std): The standard library for building CosmWasm smart contracts. Code in this package is compiled into the smart contract.
```
Comment (ethanfrey) [inline README.md:51]: This is cosmwasm-std package (tip: look in the `Cargo.toml` file in the linked directory)

File: README.md [hunk 5]
```diff
@@ -29,16 +29,31 @@ guidelines on
...
+- [cw-storage](https://crates.io/crates/cw-storage) ([repo](https://github.com/CosmWasm/cosmwasm/tree/master/packages/storage)): Helper methods to reduce boilerplate for storing data types. Easier and more secure persistence layer.
```
Comment (ethanfrey) [inline README.md:52]: I would just link the repo like above. And it is `cosmwasm-storage`

**Pull request merged**